src/three/GLTFExtensionLoader.js

```diff
@@ -40,12 +40,16 @@ export class GLTFExtensionLoader extends LoaderBase {
 
 				}
 
-
 			}
 
 			// assume any pre-registered loader has paths configured as the user desires, but if we're making
 			// a new loader, use the working path during parse to support relative uris on other hosts
-			const resourcePath = loader.resourcePath || loader.path || this.workingPath;
+			let resourcePath = loader.resourcePath || loader.path || this.workingPath;
+			if ( ! /[\\/]$/.test( resourcePath ) && resourcePath.length ) {
```

Comment:
I ran into this issue internal to the KTX2Loader when all of the checked paths were ''.

I don't have a B3D data set to sanity check with, but I don't think it would hurt?

Reply:
Sorry I forgot to respond to this -- busy week 😁. Can we add it to the B3DMLoaer, as well?